server/db/db.go

```diff
@@ -25,7 +25,7 @@ func New(driver, dsn string) (*gorm.DB, error) {
 			return nil, err
 		}
 		log.INFO.Println("using sqlite database:", file)
-		if err := os.MkdirAll(filepath.Dir(file), os.ModePerm); err != nil {
+		if err := os.MkdirAll(filepath.Dir(file), 0700); err != nil {
```

Comment:
700 or 600? Not an expert...

Reply:
We're setting permissions on directory level. Therefore 700 is needed to keep the contents listable. Alternatively we could keep the folder open as is and set evcc.db permissions to 600 after we've created the file. But this is more code and has the same effect.

Reply:
Falls die Datei im User Folder liegt (den wir nicht anlegen) bleiben ihre Permissions damit "as-is". Wir gehen davon aus, dass der Userfolder korrekt ist.

Reply:
Yes, I've kept this change deliberately conservative to not break existing setups or make unwanted modifications.

> Does not change permissions for existing sqlite files or custom database locations where the folder already exists. Here user is responsible for securing the location.